pkg/controller/secret/secret_controller.go

```diff
@@ -200,6 +200,28 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 		{Receiver: receiverNull, MatchRE: map[string]string{"namespace": alertmanager.PDRegexLP}, Match: map[string]string{"alertname": "TargetDown"}},
 		// https://issues.redhat.com/browse/OSD-5544
 		{Receiver: receiverNull, MatchRE: map[string]string{"job_name": "^elasticsearch.*"}, Match: map[string]string{"alertname": "KubeJobFailed", "namespace": "openshift-logging"}},
+		// https://issues.redhat.com/browse/OSD-11273 - silence all elasricsearch alerts so we can handle only the ones that have extended logging support
+		// the list of alerts is pulled via
+		// ```
+		// yq .spec.groups[].rules[].alert ../managed-cluster-config/deploy/sre-prometheus/externded-logging/102-parsed-fluend.PrometheusRule.yaml | awk '{print "{Receiver: receiverNull, MatchRE: map[string]string{\"job_name\": \"^elasticsearch.*\"}, Match: map[string]string{\"alertname\": \"" $1 "\", \"namespace\": \"openshift-logging\"}}"},'
+		// # for elasticsearch
+		//  yq .spec.groups[].rules[].alert ../managed-cluster-config/deploy/sre-prometheus/externded-logging/102-parsed.PrometheusRule.yaml | sort -u | awk '{print "{Receiver: receiverNull, MatchRE: map[string]string{\"job_name\": \"^elasticsearch.*\"}, Match: map[string]string{\"alertname\": \"" $1 "\", \"namespace\": \"openshift-logging\"}},"}'
+		// ```
+		{Receiver: receiverNull, MatchRE: map[string]string{"job_name": "^elasticsearch.*"}, Match: map[string]string{"alertname": "FluentdNodeDown", "namespace": "openshift-logging"}},
```

Comment:
Just to understand this PR:

Here you are setting matchers for specific ELK stack alerts and then you are connecting them with the receiverNull to silence them, right?

Reply:
Yes, I am not sure on the regex on the left which is why I wanted an owner to tell.me if what I am doing is correct